modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -39,47 +45,75 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-172949. Make sure the edit button does not appear when the user's role is 'User employee'."
+	@description = "This is a test for LPS-180358. Verify all status changes after 'need more info' request"
 	@priority = 5
-	test EmployeeCannotEditOrganizationInAwatingApprovalStatus {
-		task ("Given the Employee user is in the Organization page") {
+	test VerifyStatusAfterRequestMoreInfo {
+		task ("Given I have a Created Organization with 'Awaiting Approval On EVP' status") {
+			EVPOrganization.goToCreateOrganization(
+				button = "Create an Organization",
+				navItem = "Organization");
+
+			EVPOrganization.fillOrganizationForm(
+				bankingInfo = "Banking Info",
+				city = "City",
+				contactEmail = "Contact Email",
+				contactName = "Contact Name",
+				contactPhone = "Contact Phone",
+				country = "Country",
+				done = "true",
+				organizationName = "Organization 5",
+				organizationSite = "Social Media Link",
+				smallDescription = "Small Description",
+				state = "State",
+				street = "Street",
+				taxId = "Tax Id",
+				zipCode = "Zip");
+		}
+
+		task ("When the Manager click in 'Need more info'") {
 			User.logoutAndLoginPG(
-				userLoginEmailAddress = "employee@liferay.com",
-				userLoginFullName = "Employee User");
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "Manager User");
 
 			EVPUtils.gotoEVP();
 
 			EVPNavigator.goToNavItem(navItem = "Organization");
-		}
 
-		task ("When the Employee user clicks on ellipsis button of an Organization with 'Awaiting Finance Approval' status") {
-			EVPOrganization.kebabMenuOrganization(organizationName = "Organization 2");
+			EVPOrganization.kebabMenuOrganization(organizationName = "Organization 5");
+
+			MenuItem.click(menuItem = "Review");
+
+			EVPOrganization.changeStatus(
+				button = "Request more Info",
+				typeMessage = "Good Morning, Good Afternoon and Good Night");
 		}
 
-		task ("Then the edit button dont appear") {
-			MenuItem.viewNotPresent(menuItem = "Edit");
+		task ("Then The Organization status will change for 'Awaiting More Info From Employee'") {
+			EVPOrganization.viewStatus(
+				organizationName = "Organization 5",
+				status = "Awaiting More Info From Employee");
 		}
-	}
 
-	@description = "This is a test for LPS-172949. Make sure the edit button does not appear when the user's role is 'User manager'."
-	@priority = 5
-	test ManagerCannotEditOrganizationInAwatingApprovalStatus {
-		task ("Given the Manager user is in the Organization page") {
+		task ("When the Employee edit an Organization that is with 'Awaiting More Info From Employee' status") {
```

Comment:
Hey @juliosilveira77 , It's a good idea to inform in the task description of the user change process and his edit journey